lighthouse-core/gather/gather-runner.js

```diff
@@ -234,13 +252,20 @@ class GatherRunner {
       .then(_ => recordTrace && driver.beginTrace(options.flags))
       // Navigate.
       .then(_ => GatherRunner.loadPage(driver, options))
-      .then(_ => log.log('statusEnd', status));
+      .then(_ => log.timeEnd(status, 'log'));
```

Comment:
the `level` is log? that's weird haha

Reply:
shrug. i suppose its equivalent level is actually "info"/"informational", but that's otherwise hidden as an implementation detail of lighthouse-logger.